src/Governance.sol

```diff
@@ -112,10 +113,23 @@ contract Governance is Multicall, UserProxyFactory, ReentrancyGuard, IGovernance
         EPOCH_DURATION = _config.epochDuration;
         require(_config.epochVotingCutoff < _config.epochDuration, "Gov: epoch-voting-cutoff-gt-epoch-duration");
         EPOCH_VOTING_CUTOFF = _config.epochVotingCutoff;
+
+        if (_initiatives.length > 0) {
+            registerInitialInitiatives(_initiatives);
+        }
+    }
+
+    function registerInitialInitiatives(address[] memory _initiatives) public {
+        require(!initialInitiativesRegistered, "Initial inintiatives already registered");
+
         for (uint256 i = 0; i < _initiatives.length; i++) {
             initiativeStates[_initiatives[i]] = InitiativeState(0, 0, 0, 0, 0);
             registeredInitiatives[_initiatives[i]] = 1;
```

Comment:
I wonder what happens if we theoretically neglect to call this until the first few epochs pass... obviously, let's not do that, but still, have we thought about the consequences?

Reply:
Just want to flag that I had edited the deploy script to allow voting on `registeredInitiatives` the way to allow that was this:
https://github.com/liquity/V2-gov/blob/549877d976781b390207e0faacbc567a614c9681/script/DeploySepolia.s.sol#L95-L96

If you alter the logic, you may have to retest with something like this test:
https://github.com/liquity/V2-gov/blob/549877d976781b390207e0faacbc567a614c9681/test/E2E.t.sol#L100

Reply:
I’ll fix it.

Reply:
@GalloDaSballo I don't get this...

`epochStart` is `block.timestamp - VESTING_EPOCH_START`, but `VESTING_EPOCH_START` is:
https://github.com/liquity/V2-gov/blob/549877d976781b390207e0faacbc567a614c9681/script/DeploySepolia.s.sol#L46

That doesn't look right.

Reply:
Sorry, the intended setup is this one:
https://github.com/liquity/V2-gov/blob/549877d976781b390207e0faacbc567a614c9681/test/E2E.t.sol#L91

By setting the start one epoch in the past, the initiatives are votable

That's because we enforce only one FSM invariant and have no edge case for initiatives deployed at time zero
So by setting the start one week in the past, the initially registered initiatives are votable since the start